invenio_communities/views/ui.py

```diff
@@ -257,7 +257,7 @@ def new():
                 community = None
 
         if community:
-            permissions = current_permission_factory.keys()
+            permissions = list(current_permission_factory.keys())
```

Comment:
here we can directly do `list(current_permission_factory)` that gives a list of keys for pyton 2 and 3 :)